backend/clubs/urls.py

```diff
@@ -75,6 +76,7 @@
     path(r"settings/zoom/meetings/", MeetingZoomAPIView.as_view(), name="users-zoom-meeting"),
     path(r"settings/permissions/", UserPermissionAPIView.as_view(), name="users-permission"),
     path(r"clubs/<slug:club_code>/invite/", MassInviteAPIView.as_view(), name="club-invite"),
+    path(r"settings/calendar_url", UserUUIDAPIView.as_view(), name="user-uuid"),
```

Comment:
@rohan-gupta8 Add a slash to the end of this URL! I think Django will complain with a warning otherwise.

Reply:
Ok. Are we good to merge otherwise?